app/src/main/java/com/hover/stax/transfers/TransferFragment.kt

```diff
@@ -192,6 +204,12 @@ class TransferFragment : AbstractFormFragment(), ActionSelect.HighlightListener,
             }
         }
     }
+    private fun observeContact() {
+        transferViewModel.contact.observe(viewLifecycleOwner){
+            Timber.i("contact is ${it?.shortName()} at observing level")
```

Comment:
@davkutalek something is weird at this point. Need some help why.

How to reproduce.
Click on Try again the first time - logger shows expected value.
Click back and click on try again subsequently - contact shows null, even though at the point where it updates the contact live data is not null.

This seems to also be the bug causing.
-contact name showing as null in the transaction history after loading it subsequently.